sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -1050,55 +1053,69 @@ public String getDefaultLanguage() {
     }
 
     /**
-     * Cancel a long-running operation healthcare task by given a healthcare task identification number.
+     * Analyze actions, such as, entities recognition, PII entities recognition and key phrases extraction in a list of
+     * {@link String document} with provided request options.
      *
-     * Note: In order to use this functionality, request to access public preview is required.
-     * Azure Active Directory (AAD) is not currently supported. For more information see
-     * <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/how-tos/text-analytics-for-health?tabs=ner#request-access-to-the-public-preview">this</a>.
+     * See <a href="https://aka.ms/talangs">this</a> supported languages in Text Analytics API.
      *
      * <p><strong>Code Sample</strong></p>
-     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsAsyncClient.beginCancelHealthcareTask#String-RecognizeHealthcareEntityOptions}
+     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsAsyncClient.beginAnalyzeBatchActions#Iterable-TextAnalyticsActions-String-AnalyzeBatchActionsOptions}
      *
-     * @param healthcareTaskId The healthcare task identification number.
-     * @param options The additional configurable {@link RecognizeHealthcareEntityOptions options} that may be passed
-     * when cancelling healthcare task.
+     * @param documents A list of documents to be analyzed.
+     * For text length limits, maximum batch size, and supported text encoding, see
+     * <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits">data limits</a>.
+     * @param actions A {@link TextAnalyticsActions actions} that contains all actions to be analyzed.
```

Comment:
We should improve this javadoc. It is not clear what actions are supposed to mean.